### ISSUE TITLE
stable-diffusion-webui-tensorRT extensions convert .onnx file to .trt file fail

### ISSUE THREAD

**Issue by zzzliuke** [NVIDIA/TensorRT#3229]
## Description
D:\python_workspace\stable-diffusion-webui>"D:\python_workspace\stable-diffusion-webui\extensions\stable-diffusion-webui-tensorrt\TensorRT-8.6.1.6\bin\trtexec.exe" --onnx="D:\python_workspace\stable-diffusion-webui\models\Unet-onnx\majicmix-Realistic_betterV2V25.onnx" --saveEngine="D:\python_workspace\stable-diffusion-webui\models\Unet-trt\majicmix-Realistic_betterV2V25.trt" --minShapes=x:2x4x64x64,context:2x77x768,timesteps:2 --maxShapes=x:2x4x64x64,context:2x77x768,timesteps:2 --fp16 --workspace=8192
&&&& RUNNING TensorRT.trtexec [TensorRT v8601] # D:\python_workspace\stable-diffusion-webui\extensions\stable-diffusion-webui-tensorrt\TensorRT-8.6.1.6\bin\trtexec.exe --onnx=D:\python_workspace\stable-diffusion-webui\models\Unet-onnx\majicmix-Realistic_betterV2V25.onnx --saveEngine=D:\python_workspace\stable-diffusion-webui\models\Unet-trt\majicmix-Realistic_betterV2V25.trt --minShapes=x:2x4x64x64,context:2x77x768,timesteps:2 --maxShapes=x:2x4x64x64,context:2x77x768,timesteps:2 --fp16 --workspace=8192
[08/17/2023-11:49:40] [W] maxShapes is being broadcasted to optShapes for tensor timesteps
[08/17/2023-11:49:40] [W] maxShapes is being broadcasted to optShapes for tensor x
[08/17/2023-11:49:40] [W] maxShapes is being broadcasted to optShapes for tensor context
[08/17/2023-11:49:40] [W] --workspace flag has been deprecated by --memPoolSize flag.
[08/17/2023-11:49:40] [I] === Model Options ===
[08/17/2023-11:49:40] [I] Format: ONNX
[08/17/2023-11:49:40] [I] Model: D:\python_workspace\stable-diffusion-webui\models\Unet-onnx\majicmix-Realistic_betterV2V25.onnx
[08/17/2023-11:49:40] [I] Output:
[08/17/2023-11:49:40] [I] === Build Options ===
[08/17/2023-11:49:40] [I] Max batch: explicit batch
[08/17/2023-11:49:40] [I] Memory Pools: workspace: 8192 MiB, dlaSRAM: default, dlaLocalDRAM: default, dlaGlobalDRAM: default
[08/17/2023-11:49:40] [I] minTiming: 1
[08/17/2023-11:49:40] [I] avgTiming: 8
[08/17/2023-11:49:40] [I] Precision: FP32+FP16
[08/17/2023-11:49:40] [I] LayerPrecisions:
[08/17/2023-11:49:40] [I] Layer Device Types:
[08/17/2023-11:49:40] [I] Calibration:
[08/17/2023-11:49:40] [I] Refit: Disabled
[08/17/2023-11:49:40] [I] Version Compatible: Disabled
[08/17/2023-11:49:40] [I] TensorRT runtime: full
[08/17/2023-11:49:40] [I] Lean DLL Path:
[08/17/2023-11:49:40] [I] Tempfile Controls: { in_memory: allow, temporary: allow }
[08/17/2023-11:49:40] [I] Exclude Lean Runtime: Disabled
[08/17/2023-11:49:40] [I] Sparsity: Disabled
[08/17/2023-11:49:40] [I] Safe mode: Disabled
[08/17/2023-11:49:40] [I] Build DLA standalone loadable: Disabled
[08/17/2023-11:49:40] [I] Allow GPU fallback for DLA: Disabled
[08/17/2023-11:49:40] [I] DirectIO mode: Disabled
[08/17/2023-11:49:40] [I] Restricted mode: Disabled
[08/17/2023-11:49:40] [I] Skip inference: Disabled
[08/17/2023-11:49:40] [I] Save engine: D:\python_workspace\stable-diffusion-webui\models\Unet-trt\majicmix-Realistic_betterV2V25.trt
[08/17/2023-11:49:40] [I] Load engine:
[08/17/2023-11:49:40] [I] Profiling verbosity: 0
[08/17/2023-11:49:40] [I] Tactic sources: Using default tactic sources
[08/17/2023-11:49:40] [I] timingCacheMode: local
[08/17/2023-11:49:40] [I] timingCacheFile:
[08/17/2023-11:49:40] [I] Heuristic: Disabled
[08/17/2023-11:49:40] [I] Preview Features: Use default preview flags.
[08/17/2023-11:49:40] [I] MaxAuxStreams: -1
[08/17/2023-11:49:40] [I] BuilderOptimizationLevel: -1
[08/17/2023-11:49:40] [I] Input(s)s format: fp32:CHW
[08/17/2023-11:49:40] [I] Output(s)s format: fp32:CHW
[08/17/2023-11:49:40] [I] Input build shape: timesteps=2+2+2
[08/17/2023-11:49:40] [I] Input build shape: x=2x4x64x64+2x4x64x64+2x4x64x64
[08/17/2023-11:49:40] [I] Input build shape: context=2x77x768+2x77x768+2x77x768
[08/17/2023-11:49:40] [I] Input calibration shapes: model
[08/17/2023-11:49:40] [I] === System Options ===
[08/17/2023-11:49:40] [I] Device: 0
[08/17/2023-11:49:40] [I] DLACore:
[08/17/2023-11:49:40] [I] Plugins:
[08/17/2023-11:49:40] [I] setPluginsToSerialize:
[08/17/2023-11:49:40] [I] dynamicPlugins:
[08/17/2023-11:49:40] [I] ignoreParsedPluginLibs: 0
[08/17/2023-11:49:40] [I]
[08/17/2023-11:49:40] [I] === Inference Options ===
[08/17/2023-11:49:40] [I] Batch: Explicit
[08/17/2023-11:49:40] [I] Input inference shape: x=2x4x64x64
[08/17/2023-11:49:40] [I] Input inference shape: timesteps=2
[08/17/2023-11:49:40] [I] Input inference shape: context=2x77x768
[08/17/2023-11:49:40] [I] Iterations: 10
[08/17/2023-11:49:40] [I] Duration: 3s (+ 200ms warm up)
[08/17/2023-11:49:40] [I] Sleep time: 0ms
[08/17/2023-11:49:40] [I] Idle time: 0ms
[08/17/2023-11:49:40] [I] Inference Streams: 1
[08/17/2023-11:49:40] [I] ExposeDMA: Disabled
[08/17/2023-11:49:40] [I] Data transfers: Enabled
[08/17/2023-11:49:40] [I] Spin-wait: Disabled
[08/17/2023-11:49:40] [I] Multithreading: Disabled
[08/17/2023-11:49:40] [I] CUDA Graph: Disabled
[08/17/2023-11:49:40] [I] Separate profiling: Disabled
[08/17/2023-11:49:40] [I] Time Deserialize: Disabled
[08/17/2023-11:49:40] [I] Time Refit: Disabled
[08/17/2023-11:49:40] [I] NVTX verbosity: 0
[08/17/2023-11:49:40] [I] Persistent Cache Ratio: 0
[08/17/2023-11:49:40] [I] Inputs:
[08/17/2023-11:49:40] [I] === Reporting Options ===
[08/17/2023-11:49:40] [I] Verbose: Disabled
[08/17/2023-11:49:40] [I] Averages: 10 inferences
[08/17/2023-11:49:40] [I] Percentiles: 90,95,99
[08/17/2023-11:49:40] [I] Dump refittable layers:Disabled
[08/17/2023-11:49:40] [I] Dump output: Disabled
[08/17/2023-11:49:40] [I] Profile: Disabled
[08/17/2023-11:49:40] [I] Export timing to JSON file:
[08/17/2023-11:49:40] [I] Export output to JSON file:
[08/17/2023-11:49:40] [I] Export profile to JSON file:
[08/17/2023-11:49:40] [I]
[08/17/2023-11:49:41] [I] === Device Information ===
[08/17/2023-11:49:41] [I] Selected Device: Quadro T2000
[08/17/2023-11:49:41] [I] Compute Capability: 7.5
[08/17/2023-11:49:41] [I] SMs: 16
[08/17/2023-11:49:41] [I] Device Global Memory: 4095 MiB
[08/17/2023-11:49:41] [I] Shared Memory per SM: 64 KiB
[08/17/2023-11:49:41] [I] Memory Bus Width: 128 bits (ECC disabled)
[08/17/2023-11:49:41] [I] Application Compute Clock Rate: 1.785 GHz
[08/17/2023-11:49:41] [I] Application Memory Clock Rate: 6.001 GHz
[08/17/2023-11:49:41] [I]
[08/17/2023-11:49:41] [I] Note: The application clock rates do not reflect the actual clock rates that the GPU is currently running at.
[08/17/2023-11:49:41] [I]
[08/17/2023-11:49:41] [I] TensorRT version: 8.6.1
[08/17/2023-11:49:41] [I] Loading standard plugins
[08/17/2023-11:49:41] [I] [TRT] [MemUsageChange] Init CUDA: CPU +301, GPU +0, now: CPU 6998, GPU 855 (MiB)
[08/17/2023-11:49:46] [I] [TRT] [MemUsageChange] Init builder kernel library: CPU +743, GPU +172, now: CPU 8874, GPU 1027 (MiB)
[08/17/2023-11:49:46] [W] [TRT] CUDA lazy loading is not enabled. Enabling it can significantly reduce device memory usage and speed up TensorRT initialization. See "Lazy Loading" section of CUDA documentation https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#lazy-loading
[08/17/2023-11:49:46] [I] Start parsing network model.
[libprotobuf WARNING **************************************************************************\externals\protobuf\3.0.0\src\google\protobuf\io\coded_stream.cc:604] Reading dangerously large protocol message.  If the message turns out to be larger than 2147483647 bytes, parsing will be halted for security reasons.  To increase the limit (or to disable these warnings), see CodedInputStream::SetTotalBytesLimit() in google/protobuf/io/coded_stream.h.
[libprotobuf WARNING **************************************************************************\externals\protobuf\3.0.0\src\google\protobuf\io\coded_stream.cc:81] The total number of bytes read was 1721624086
[08/17/2023-11:49:47] [I] [TRT] ----------------------------------------------------------------
[08/17/2023-11:49:47] [I] [TRT] Input filename:   D:\python_workspace\stable-diffusion-webui\models\Unet-onnx\majicmix-Realistic_betterV2V25.onnx
[08/17/2023-11:49:47] [I] [TRT] ONNX IR version:  0.0.8
[08/17/2023-11:49:47] [I] [TRT] Opset version:    17
[08/17/2023-11:49:47] [I] [TRT] Producer name:    pytorch
[08/17/2023-11:49:47] [I] [TRT] Producer version: 2.0.1
[08/17/2023-11:49:47] [I] [TRT] Domain:
[08/17/2023-11:49:47] [I] [TRT] Model version:    0
[08/17/2023-11:49:47] [I] [TRT] Doc string:
[08/17/2023-11:49:47] [I] [TRT] ----------------------------------------------------------------
[libprotobuf WARNING **************************************************************************\externals\protobuf\3.0.0\src\google\protobuf\io\coded_stream.cc:604] Reading dangerously large protocol message.  If the message turns out to be larger than 2147483647 bytes, parsing will be halted for security reasons.  To increase the limit (or to disable these warnings), see CodedInputStream::SetTotalBytesLimit() in google/protobuf/io/coded_stream.h.
[libprotobuf WARNING **************************************************************************\externals\protobuf\3.0.0\src\google\protobuf\io\coded_stream.cc:81] The total number of bytes read was 1721624086
[08/17/2023-11:49:48] [W] [TRT] onnx2trt_utils.cpp:374: Your ONNX model has been generated with INT64 weights, while TensorRT does not natively support INT64. Attempting to cast down to INT32.
[08/17/2023-11:49:50] [I] Finished parsing network model. Parse time: 3.49453
[08/17/2023-11:49:50] [I] [TRT] BuilderFlag::kTF32 is set but hardware does not support TF32. Disabling TF32.
[08/17/2023-11:49:50] [I] [TRT] Graph optimization time: 0.472322 seconds.
[08/17/2023-11:49:50] [I] [TRT] [MemUsageChange] Init cuBLAS/cuBLASLt: CPU +0, GPU +8, now: CPU 11333, GPU 2411 (MiB)
[08/17/2023-11:49:50] [I] [TRT] [MemUsageChange] Init cuDNN: CPU +0, GPU +10, now: CPU 11333, GPU 2421 (MiB)
[08/17/2023-11:49:51] [W] [TRT] TensorRT was linked against cuDNN 8.9.0 but loaded cuDNN 8.7.0
[08/17/2023-11:49:51] [I] [TRT] BuilderFlag::kTF32 is set but hardware does not support TF32. Disabling TF32.
[08/17/2023-11:49:51] [I] [TRT] Local timing cache in use. Profiling results in this builder pass will not be stored.
[08/17/2023-11:52:17] [E] Error[10]: Could not find any implementation for node {ForeignNode[onnx::LayerNormalization_10453 + (Unnamed Layer* 175) [Shuffle].../input_blocks.1/input_blocks.1.1/Reshape_2 + /input_blocks.1/input_blocks.1.1/Transpose_1 + /input_blocks.1/input_blocks.1.1/Reshape_3]}.
[08/17/2023-11:52:18] [E] Error[10]: [optimizer.cpp::nvinfer1::builder::cgraph::LeafCNode::computeCosts::3869] Error Code 10: Internal Error (Could not find any implementation for node {ForeignNode[onnx::LayerNormalization_10453 + (Unnamed Layer* 175) [Shuffle].../input_blocks.1/input_blocks.1.1/Reshape_2 + /input_blocks.1/input_blocks.1.1/Transpose_1 + /input_blocks.1/input_blocks.1.1/Reshape_3]}.)
[08/17/2023-11:52:18] [E] Engine could not be created from network
[08/17/2023-11:52:18] [E] Building engine failed
[08/17/2023-11:52:18] [E] Failed to create engine from model or file.
[08/17/2023-11:52:18] [E] Engine set up failed
&&&& FAILED TensorRT.trtexec [TensorRT v8601] # D:\python_workspace\stable-diffusion-webui\extensions\stable-diffusion-webui-tensorrt\TensorRT-8.6.1.6\bin\trtexec.exe --onnx=D:\python_workspace\stable-diffusion-webui\models\Unet-onnx\majicmix-Realistic_betterV2V25.onnx --saveEngine=D:\python_workspace\stable-diffusion-webui\models\Unet-trt\majicmix-Realistic_betterV2V25.trt --minShapes=x:2x4x64x64,context:2x77x768,timesteps:2 --maxShapes=x:2x4x64x64,context:2x77x768,timesteps:2 --fp16 --workspace=8192

## Environment
window 10 professional

**TensorRT Version**:

**NVIDIA GPU**:

**NVIDIA Driver Version**:

**CUDA Version**:

**CUDNN Version**:

Operating System:

Python Version (if applicable):

Tensorflow Version (if applicable):

PyTorch Version (if applicable):

Baremetal or Container (if so, version):

## Relevant Files

**Model link**:

## Steps To Reproduce

**Commands or scripts**:

**Have you tried [the latest release](https://developer.nvidia.com/tensorrt)?**:

**Can this model run on other frameworks?** For example run ONNX model with ONNXRuntime (`polygraphy run <model.onnx> --onnxrt`):

**Comment by zzzliuke** [comment]
TensorRT Version:TensorRT-8.6.1.6

**Comment by zerollzeng** [comment]
> __[08/17/2023-11:52:17] [E] Error[10]: Could not find any implementation for node {ForeignNode[onnx::LayerNormalization_10453 + (Unnamed Layer_ 175) [Shuffle].../input_blocks.1/input_blocks.1.1/Reshape_2 + /input_blocks.1/input_blocks.1.1/Transpose_1 + /input_blocks.1/input_blocks.1.1/Reshape_3]}. [08/17/2023-11:52:18] [E] Error[10]: [optimizer.cpp::nvinfer1::builder::cgraph::LeafCNode::computeCosts::3869] Error Code 10: Internal Error (Could not find any implementation for node {ForeignNode[onnx::LayerNormalization_10453 + (Unnamed Layer_ 175) [Shuffle].../input_blocks.1/input_blocks.1.1/Reshape_2 + /input_blocks.1/input_blocks.1.1/Transpose_1 + /input_blocks.1/input_blocks.1.1/Reshape_3]}.)

Insufficient memory? cc @nvpohanh

**Comment by nvpohanh** [comment]
@zzzliuke Could you add "--verbose" flag to show the verbose logs? Thanks

**Comment by CUGfred** [comment]
![image](https://github.com/NVIDIA/TensorRT/assets/32841220/da12aaf8-d598-4a29-ba2f-957f1490f251)
i met same issue , here is a log short, i've added --verbose flag.

**Comment by nvpohanh** [comment]
@CUGfred Your issue looks different. Which TRT version is it?

**Comment by CUGfred** [comment]
@nvpohanh TensorRT 8.6.1.6 just download tar file, and extract into stable-diffusion-webui-tensorRT folder, i tested unet onnx(opset 17) in onnxruntime it could get output. while i use polygraphy to test(opset 12), it failed.

**Comment by zerollzeng** [comment]
The "Segmentation fault" error looks like a bug to me. @CUGfred Could you please share the onnx model with us? We can take a further check.

**Comment by CUGfred** [comment]
yeah im glad to share, onnx model is from stable-diffusion-webui-tensorrt extension. it export onnx from dreamshaperv7 safetensor,  if you need that file, please reply me and i will find how to upload safetensor.
![image](https://github.com/NVIDIA/TensorRT/assets/32841220/6408045a-db4f-4803-a132-c62299fae28e)
![image](https://github.com/NVIDIA/TensorRT/assets/32841220/1aafd335-4871-4fa3-946a-7971f2250e75)
yesterday i tried again, onnx export use opset 14, tensorrt could convert in webui, while i use trtexec to load, it failed. tensorrt convert engine instruction included in the first image. in addition i use  nvcr.io/nvidia/pytorch:22.09-py3 image to run a container. python version is 3.8

**Comment by CUGfred** [comment]
@zerollzeng today, i have tested another docker image nvcr.io/nvidia/pytorch:23.06-py3, which got it done. engine generated and inference done. bugs i posted is caused by the tensorrt version(maybe)

**Comment by zerollzeng** [comment]
Glad to know, can we close the issue? Thanks!

**Comment by ttyio** [comment]
Closing since no activity for more than 3 weeks, pls reopen if you still have question, thanks!
